Tools/TestWebKitAPI/mac/mainMac.mm

```diff
@@ -47,6 +58,8 @@ void buildArgumentDefaults(int argc, char** argv, NSMutableDictionary *argumentD
             argumentDefaults[@"WebKit2GPUProcessForDOMRendering"] = @YES;
         else if (strcmp(argv[i], "--no-use-gpu-process") == 0)
             argumentDefaults[@"WebKit2GPUProcessForDOMRendering"] = @NO;
+        else if (strcmp(argv[i], "--force-site-isolation") == 0)
```

Comment:
--site-isolation instead of --force-site-isolation would be slightly preferable to me as it matches the run-webkit-tests argument.